File: core/src/main/java/org/toml/ide/formatter/settings/TomlCodeStyleMainPanel.java
/*
 * Use of this source code is governed by the MIT license that can be
 * found in the LICENSE file.
 */

package org.toml.ide.formatter.settings;

import consulo.language.codeStyle.CodeStyleSettings;
import consulo.language.codeStyle.ui.setting.TabbedLanguageCodeStylePanel;
import jakarta.annotation.Nonnull;
import org.toml.lang.TomlLanguage;

public class TomlCodeStyleMainPanel extends TabbedLanguageCodeStylePanel {
    public TomlCodeStyleMainPanel(@Nonnull CodeStyleSettings currentSettings, @Nonnull CodeStyleSettings settings) {
        super(TomlLanguage.INSTANCE, currentSettings, settings);
    }

    @Override
    protected void initTabs(@Nonnull CodeStyleSettings settings) {
        addIndentOptionsTab(settings);
    }
}

File: core/src/main/kotlin/org/toml/ide/formatter/settings/TomlCodeStyleMainPanel.kt
/*
 * Use of this source code is governed by the MIT license that can be
 * found in the LICENSE file.
 */

package org.toml.ide.formatter.settings

import consulo.language.codeStyle.CodeStyleSettings
import consulo.language.codeStyle.ui.setting.TabbedLanguageCodeStylePanel
import org.toml.lang.TomlLanguage

class TomlCodeStyleMainPanel(currentSettings: CodeStyleSettings, settings: CodeStyleSettings) :
    TabbedLanguageCodeStylePanel(TomlLanguage, currentSettings, settings) {

    override fun initTabs(settings: CodeStyleSettings) {
        addIndentOptionsTab(settings)
    }
}
